website/content/tutorials/1-getting-started/_index.md

```diff
@@ -0,0 +1,954 @@
+                <description>The type of ATA socket on the motherboard , <code>pata</code> (parallel ATA), <code>sata</code> (Serial ATA), or an alternative.</description>
+              </define-field>
+            </model>
+          </define-assembly>
+          <define-assembly name="memory" min-occurs="1" max-occurs="unbounded">
+            <formal-name>Motherboard Random Access Memory (RAM) Module(s)</formal-name>
+            <description>Random access memory hardware installed on the motherboard of a computer.</description>
+            <group-as name="memory-modules" in-json="ARRAY"/>
+            <model>
+              <assembly ref="vendor"/>
+              <field ref="product-name"/>
+              <define-field name="byte-size" as-type="positiveInteger" min-occurs="1" max-occurs="1">
+                <formal-name>Memory Module Size</formal-name>
+                <description>Size of the memory module in binary, not SI base-10 units, meaning a kilobyte is 1024 bytes, not 1000 bytes.</description>
+              </define-field>
+            </model>
+          </define-assembly>
+          <define-assembly name="expansion-card" min-occurs="0" max-occurs="unbounded">
+            <formal-name>Motherboard Expansion Card</formal-name>
+            <description>The model number of an expansion card connected to the motherboard of a computer.</description>
+            <group-as name="expansion-cards" in-json="ARRAY"/>
+            <model>
+              <assembly ref="vendor"/>
+              <define-field name="product-name" as-type="string" min-occurs="1" max-occurs="1">
+                <formal-name>Product Name</formal-name>
+                <description>The product name from the vendor of the computer part.</description>
+              </define-field>
+              <define-field name="type" as-type="string" min-occurs="1" max-occurs="1">
+                <formal-name>Expansion Card Type</formal-name>
+                <description>The type of expansion card on a motherboard of a computer, such as <code>pci</code> (PCI, e.g. Peripheral Component Interconnect), <code>pcie</code> (PCI Express), or an alternative.</description>
+              </define-field>
+            </model>
+          </define-assembly>
+        </model>
+      </define-assembly>
+    </model>
+  </define-assembly>
+</METASCHEMA>
+```
+
+We lifted the `assembly` definition for vendor and the definition of the product name `field` to outside the computer `assembly`. Because we have a `root-name` previously defined for the computer `assembly`, Metaschema enabled-tools will work just like before, generating and parsing the same instances with the computer `assembly` as the root, even with multiple top-level elements defined. At the same time, we reduced repeat copy-pasted code, and we can continue to add other requirements from our stakeholders and reuse their definitions across different elements of the model and maintain the original definition once.
```

Comment:
s/will work just like before/will work just as before

(This "like vs as" rule is changing but I don't think it's changed completely yet.)

Reply:
```suggestion
We lifted the `assembly` definition for vendor and the definition of the product name `field` to outside the computer `assembly`. Because we have a `root-name` previously defined for the computer `assembly`, Metaschema-enabled tools will work just as before, generating and parsing the same instances with the computer `assembly` as the root, even with multiple top-level elements defined. At the same time, we reduced repeat copy-pasted code, and we can continue to add other requirements from our stakeholders and reuse their definitions across different elements of the model and maintain the original definition once.
```